docs/source/manifest_submission.rst

```diff
@@ -0,0 +1,274 @@
+Submit a manifest to Synapse
+============================
+
+Prerequisites
+-------------
+
+**Obtain Synapse Credentials**:
```

Comment:
You can probably refer to the installation guide instead of having all of this

Reply:
Yeah, I linked the installation guide.